### ISSUE TITLE
Is live stream support subtitle?

### ISSUE THREAD

**Issue by fingul** [kaltura/nginx-vod-module#1302]
Hi! 

Is live stream support subtitle?

Looks like not working.

Have a nice day!

**Comment by erankor** [comment]
Yes, subtitles are support for live, if you use DASH, you need to use `vod_dash_subtitle_format smpte-tt`
